.eslintrc.yml

```diff
@@ -58,3 +58,27 @@ rules:
 
   mocha-no-only/mocha-no-only:
     - error
+
+  "@typescript-eslint/naming-convention":
+    - error
+    - selector: variable
+      format:
+        - camelCase
+        - PascalCase
```

Comment:
I think at least Pascal can be verbotten! I can see snake for things extracted from JSON and CONSTANT_CASE for globals